app/design/frontend/Jungle-luma/jungle_theme/theme.xml

```diff
@@ -0,0 +1,10 @@
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```

Comment:
Remove copyright block add theme preview image

Reply:
Removed the comment block.